### DISCUSSION TITLE
resolution of uploaded images (with specified resolution parameters when i put it manually) and generated images does not match.

### DISCUSSION THREAD

**Discussion by vitalijmirnyj** [TheLastBen/fast-stable-diffusion#1554] [Q&A]
Hello. I am not sure if this is something wrong on my side, and i am sorry if it is, but i wanted to share. The problem i found is in img2img generated images resolutions. For example if i put image in resolution 464x526 and specify exact resolution in parameters the generated image is a little smaller, just a few pixels in both axes, so the resolution of uploaded images (with specified resolution parameters when i put it manually) and generated images does not match.

**Comment by TheLastBen** [comment]
Check with the A1111 repo for that issue https://github.com/AUTOMATIC1111/stable-diffusion-webui
